examples/predictors/lstm/tests/test_xprize_predictor.py

```diff
@@ -10,15 +10,10 @@
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
This test and its fixtures are only for the LSTM example. So it belongs to the LSTM folder.